tests/UI/pages/BO/design/linkWidgets/index.js

```diff
@@ -32,26 +32,26 @@ module.exports = class LinkWidgets extends BOBasePage {
   /* Table methods */
   /**
    * Get Number of element in grid
-   * @param hookId, table to get number from
+   * @param hookName, table to get number from
```

Comment:
What is `table` here ?

Reply:
this one 
![image](https://user-images.githubusercontent.com/48441421/87677593-3e193b80-c77a-11ea-965b-2b82518d8dcc.png)